src/integration-tests/model-in-image/README

```diff
@@ -0,0 +1,120 @@
+Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+SUMMARY:
+
+  This is a stand alone test for running the mii-sample in
+  'kubernetes/samples/scripts/create-weblogic-domain/model-in-image'.
+
+IMPORTANT NOTES:
+
+  (1) This test isn't currently integrated with the test
+      suite framework. Instructions for doing so are below.
+
+  (2) This test assumes Kubernetes is running locally 
+      on a single machine. (It builds/uses local images
+      without pushing them.)
+
+-------------------
+To run stand alone:
+-------------------
+
+  1) Optionally customize the following:
+
+     export WORKDIR=/tmp/$USER/model-in-image-sample-work-dir
+     export MODEL_IMAGE_NAME=model-in-image
+     export MODEL_IMAGE_TAG=v1
+     export DOMAIN_NAMESPACE=sample-domain1-ns
+     export DOMAIN_UID=sample-domain1
+
+     Defaults are as shown.
+
+     See the sample's 'custom-env.sh' file and this
+     test's 'env.sh' for additional customizable env vars.
+
+  2) If running JRF:
+
+     export DB_NAMESPACE=
+
+     This defaults to 'default'.
+
+  3) Run the test.
+
+     For WLS: ./run.sh -clean [-noupdate]
+     For JRF: ./run.sh -clean [-noupdate] -jrf -db
+
+     WARNINGS AND NOTES:
+
+       - The '-clean' option:
+
+         - Calls 'cleanup.sh' before doing anything else, 
+           which deletes almost everything in your k8s cluster.
+
+         - Deletes the current WORKDIR before proceeding.
+
+         - Implicitly cause the test to build/deploy the operator,
+           and to deploy traefik on node port 303035.
+
+       - The '-db' option will cause the test to deploy Oracle DB.
+
+       - Pass '-noupdate' to skip the multi-minute 'update' portion of the test.
```

Comment:
What's the meaning of multi-minute update?

Reply:
Note that this directory is for test purposes, it's not part of the sample.

The reason for the comment is that it takes several minutes to do a roll.  So the option is there to skip that part.  When testing, or giving a 'simplified' form of the demo, it's often sufficient to skip the roll demo...